ReportManagement/META-INF/MANIFEST.MF

```diff
@@ -8,3 +8,4 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Automatic-Module-Name: ReportManagement
 Import-Package: org.osgi.framework;version="1.3.0"
 Bundle-ActivationPolicy: lazy
+Require-Bundle: HMS_DB
```

Comment:
This should not be a required bundle. Change it to Import-package
`Import-package - hms_db`